swagger/swagger-ui/oauth2-redirect.html

```diff
@@ -0,0 +1,72 @@
+<!doctype html>
+<html lang="en-US">
+<title>Swagger UI: OAuth2 Redirect</title>
+<body>
+</body>
+</html>
+<script>
+    'use strict';
+    function run () {
+        var oauth2 = window.opener.swaggerUIRedirectOauth2;
+        var sentState = oauth2.state;
+        var redirectUrl = oauth2.redirectUrl;
+        var isValid, qp, arr;
+
+        if (/code|token|error/.test(window.location.hash)) {
+            qp = window.location.hash.substring(1);
+        } else {
+            qp = location.search.substring(1);
+        }
+
+        arr = qp.split("&")
+        arr.forEach(function (v,i,_arr) { _arr[i] = '"' + v.replace('=', '":"') + '"';})
```

Comment:
More of an informative comment for myself, do we need to escape the `"` in

```javascript
 { _arr[i] = '"' + v.replace('=', '":"') + '"';})
```
or will it work without?

Reply:
Ah I see, I suppose then it works without escaping! TIL

Reply:
@mateiidavid I too hope it works as intended, lol